src/celeritas/phys/PhysicsData.hh

```diff
@@ -179,6 +180,11 @@ struct HardwiredModels
     ModelId chips;
     NeutronElasticData<W, M> chips_data;
 
+    // Neutron inelastic
+    ProcessId neutron_inelastic;
+    ModelId bertini;
+    NeutronInelasticData<W, M> bertini_data;
+
```

Comment:
It's definitely time to support models using independent kernels to generate their own cross sections... this is growing out of hand 😓